docs/troubleshooting.md

```diff
@@ -19,4 +20,16 @@ Notice: Undefined index: X-Pantheon-Styx-Hostname in /var/www/html/vendor/panthe
  [error]  Pantheon headers missing, which is not quite right.
 ```
 
-The `test` enviroment has not been created. Dev needs promoted first before the files or database can be pulled from it.
\ No newline at end of file
+The `test` enviroment has not been created. Dev needs promoted first before the files or database can be pulled from it.
+
+## Lando won't build due to a missing ssh file
+
+What the error looks like:
+
+```
+Error =⇒ ENOENT: no such file or directory, scandir /Users/<your_username>/.ssh
+```
+
+Solution: 
+
+This error occurs when you haven't generated an SSH key on your machine. Following Pantheon's documentation on [how to generate a SSH key](https://pantheon.io/docs/ssh-keys#generate-ssh-key) will resolve this issue. 
```

Comment:
@dayvod I think it would be helpful to add some more context if possible or clarifying the situation where this could happen. For example, the word `build` is vague and unclear. Imagine the person running into the issue isn't familiar with Lando at all. The user isn't directly running a command like `lando build`. Is this when booting the application locally ie `lando start`. Is it when _building_ the app for deployment?
Something to think about. Definitely a helpful contribution though. Thanks for adding!